docs/whats_new.rst

```diff
@@ -26,7 +26,7 @@ Bugfixes
   types requirement by `Synchon Mandal`_ (:gh:`339`)
 - Remove extra dimension from masks warped via ANTs by `Synchon Mandal`_
   (:gh:`340`)
-
+- Add 'emomatching' task to :class:`DataladAOMICPIOP2` by `Leonard Sasse`_ (:gh:`370`)
```

Comment:
This should now be in the `docs/changes/newsfragments` directory, in a file named 370.bugfix that has the following content:
```
Add ``'emomatching'`` task to :class:`DataladAOMICPIOP2` by `Leonard Sasse`_
```

Reply:
i see